### DISCUSSION TITLE
Compilation instructions out of date

### DISCUSSION THREAD

**Discussion by hlein** [aquasecurity/trivy#4509] [Documentation]
### Description

The documentation at https://aquasecurity.github.io/trivy/v0.41/getting-started/installation/#install-from-github-release-official still discusses the old way of building from source:
```
git clone --depth 1 --branch v0.41.0 https://github.com/aquasecurity/trivy
cd trivy
go install
```
This now fails with `no Go files in /path/to/trivy`

AFAIK this has not worked since 0.39.0 when `Makefile` was removed in favor of using `mage`.

### Suggestions

The new method appears to be to install `mage` as a prereq to building, and then do `mage build` from the toplevel dir.

**Comment by knqyf263** [comment]
```
go install cmd/trivy/main.go
```

should work.

**Reply by knqyf263** [reply]
`go install ./cmd/trivy` is better. I created an issue. Thanks.
https://github.com/aquasecurity/trivy/issues/4510
